src/api/application/application.ts

```diff
@@ -15,29 +15,39 @@ export class NavigationRejectedReply extends Reply<'window-navigation-rejected',
     public url: string;
 }
 
+
 export default class ApplicationModule extends Bare {
+    /**
+     * Returns an Application object that represents an existing application.
+     * @param { object } indentity
+     * @return {Application}
+    */
     public wrap(identity: Identity): Application {
         return new Application(this.wire, identity);
     }
 
+    /**
+     * @param appOptions 
+     * @return {Application}
+     */
     public create(appOptions: any): Promise<Application> {
         return this.wire.sendAction('create-application', appOptions)
             .then(() => this.wrap({ uuid: appOptions.uuid }));
     }
 }
 
 /**
-  @classdesc An object representing an application. Allows the developer to create,
-  execute, show/close an application as well as listen to application events.
-  as its argument
-  @class
+ * @classdesc An object representing an application. Allows the developer to create,
+ * execute, show/close an application as well as listen to application events.
+ * as its argument
+ * @class
 */
 export class Application extends Base {
 
     /**
-        @param { object } wire
-        @param { object } identity
-        @constructor
+     * @param { object } wire
+     * @param { object } identity
+     * @constructor
```

Comment:
@andreGarvin lets not document any of these constructors that receive a wire, these are internal use only.